src/Styleguide/Artwork/__stories__/Banner.story.tsx

```diff
@@ -3,5 +3,14 @@ import { storiesOf } from "storybook/storiesOf"
 import { Banner } from "../Banner"
 
 storiesOf("Styleguide/Artwork", module).add("Banner", () => {
-  return <Banner />
+  return (
+    <Theme>
```

Comment:
We don't need `<Theme />` any more if you `import { storiesOf } from 'storybook/storiesOf`

Reply:
Sounds good